packages/sdk-js/src/index.ts

```diff
@@ -610,9 +611,20 @@ class Client extends BaseClient {
    * @param options - Request options, including workflowId and orgId for filtering
    * @returns {Promise<ListSecretResponse[]>} - The list of secrets
    */
+  /**
+   * List secrets with pagination support
+   * @param options - Request options including pagination parameters
+   * @param options.workflowId - Filter secrets by workflow ID
+   * @param options.orgId - Filter secrets by organization ID
+   * @param options.cursor - Legacy cursor parameter (deprecated, use before/after instead)
+   * @param options.before - Get items before this cursor value (for backward pagination)
+   * @param options.after - Get items after this cursor value (for forward pagination)
+   * @param options.itemPerPage - Number of items per page
+   * @returns {Promise<ListSecretsResponse>} - The list of secrets with pagination metadata
+   */
   async listSecrets(
```

Comment:
call this function `getSecrets` to be consistent with other methods. Update test code as well.